### PR TITLE
OCPBUGS-45075: VPA OWNERS: Remove John Kyros, et al., add Max Cao

### PR DESCRIPTION
**Pull request by joelsmith**

### CODE REVIEW

**Review by maxcao13 — APPROVED** [review]
/lgtm

Just wondering, why doesn't this repo automatically approve the PR when you are the author, but the other ones do?

**Comment by joelsmith** [comment]
> Just wondering, why doesn't this repo automatically approve the PR when you are the author, but the other ones do?

Probably a bot bug. As an experiment, I'll wait to add `/approve` to see if it's just slow or if it really is waiting for my explicit approval.

**Pull request merged**
